DaisyMicheal/Ecommerce_web
# insert, delete, and update signupPage

**Pull Request Description for Signup Page**

**What does this PR do?**
Create and style the signup page for the e-commerce platform.

**Description of Task to be completed:**
- Design and implement the signup page layout, including fields for user registration (e.g., name, email, password).
- Integrate form validation to ensure users provide valid information (e.g., email format, password strength).
- Implement error handling and display user-friendly error messages for invalid inputs.
- Add a "Sign Up" button that registers the user and redirects them to the appropriate page upon successful signup.
- Include an option for users to sign up using social media accounts (e.g., Google).
- Ensure the signup page is fully responsive and accessible across different devices.
- Write unit tests to cover the form validation and user registration process.
- Securely handle user data and integrate with the backend for account creation.

**How should this be manually tested?**
- Pull the branch and install dependencies if necessary.
- Navigate to the signup page and enter valid and invalid data to test the form validation.
- Attempt to sign up using both the form fields and social media options.
- Verify that error messages are displayed for invalid inputs and that the user is redirected correctly upon successful signup.
- Run the unit tests to ensure they pass.

**Any background context you want to provide?**
None

**What are the relevant Pivotal Tracker stories?**
#188043169

**Screenshots (if appropriate):**
[image]

**Questions:**
None
